docs/building-a-form/adding-widgets-and-definitions.md

```diff
@@ -0,0 +1,208 @@
+# Adding widgets and definitions
+
+There are many common fields and widgets you can use to build forms. There are common types of definitions: `schema`/`uiSchema` objects and functions that return `schema`/`uiSchema` objects. For the function versions, there is documentation within the fields for the parameters.
+
+### In this guide
+
+- [Common widgets](#common-widgets)
+- [Common definitions](#common-definitions)
+
+### Common widgets
+
+These common widgets are included in the schemaform by default. Set these widgets in a config file while building your form. Some are associated with particular schema types or formats. There are additional widgets available, but the schemaform uses definitions they're included in, rather than just the field.
+
+To use a widget, set `ui:widget` for a field to the name of the widget. Widgets are located in [/src/js/widgets](src/js/widgets).
+
+Widget       | Default schema type or format
+------------ | -------------
+`ArrayCountWidget` |
+`CheckboxWidget` | type: `boolean`
+`CurrencyWidget` |
+`DateWidget` |
+`EmailWidget` |
+`PhoneNumberWidget`
+`RadioWidget` |
+`SelectWidget` | type: `string` with an enum property
+`SSNWidget` |
+`TextWidget` | type: `string`
+`YesNoWidget` |
+
+### Common definitions
```

Comment:
I think we're talking about doing a bit of refactoring of how definitions work, but let's go ahead with this language for now and we'll just have to update it later when the refactor is done.

Reply:
Good to know! 👍